apps/aestratum/src/aestratum.hrl

```diff
@@ -0,0 +1,95 @@
+-define(PAYMENT_CONTRACT_TESTNET_ADDRESS, <<"ct_jgo43gdpikScyo5munDvsVWBJDugjkMtqPHUfzGJyuWAFFUuB">>).
```

Comment:
http://52.10.46.160:3013/v2/accounts/ak_jgo43gdpikScyo5munDvsVWBJDugjkMtqPHUfzGJyuWAFFUuB
> {"balance":0,"id":"ak_jgo43gdpikScyo5munDvsVWBJDugjkMtqPHUfzGJyuWAFFUuB","kind":"basic","nonce":0}

http://52.10.46.160:3013/v2/contracts/ct_jgo43gdpikScyo5munDvsVWBJDugjkMtqPHUfzGJyuWAFFUuB
> {"abi_version":1,"active":true,"deposit":0,"id":"ct_jgo43gdpikScyo5munDvsVWBJDugjkMtqPHUfzGJyuWAFFUuB","owner_id":"ak_6A2vcm1Sz6aqJezkLCssUXcyZTX7X8D5UwbuS2fRJr9KkYpRU","referrer_ids":[],"vm_version":3}

Reply:
Is vm_version 3 intended? Impact?

Reply:
I can't see that the contract does anything that has changed from 3 -> 4... So should be fine.

Reply:
OK